package.json

```diff
@@ -56,6 +56,7 @@
         "verify:docs": "bash ci/verify-docs.sh"
     },
     "dependencies": {
+        "del": "^6.0.0",
```

Comment:
I'm still not fully sure we should add it. Checked `del` a bit deeper and found out that:
- the last release was 1,5 year ago - see: https://github.com/sindresorhus/del/releases
- the last commit was 1 year ago - see: https://github.com/sindresorhus/del/commits/main

That is for me a sign that the package is questionable. And looking at https://bundlephobia.com/package/del@6.0.0 it seems like minified version is 125 kB (quite big for a single purpose package).

Ref.: https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/435388485/Adding+new+dependency+to+Cloudify+UI

Until we have RD-1887 implemented I think we should keep `cloudify-ui-common` as small as possible. 

If we decide to keep `del`, then I'm also not sure if `dependencies` is the right place for it. Shouldn't it be in `peerDependencies` similarly to all cypress related deps?

Reply:
You convinced me. 125 kB it's too much for our single purpose use. I removed del and I used node native function.

However I think that del is a safe library to use. Especially for the cases of removing a lot of files and folder in complex way. There is not a lot of changes because the library is very simple and there is not much to improve. It's also very popular - has a lot of downloads https://www.npmjs.com/package/del .